build/base/Dockerfile

```diff
@@ -3,11 +3,12 @@ RUN apk add libc6-compat
 RUN apk add --no-cache git
 RUN apk add --no-cache bash
 RUN apk add --no-cache openssh
```

Comment:
I think these could be combined in to one `RUN` like below - would reduce the number of build stages

Reply:
Good catch. will be improved